cfme/services/myservice/ui.py

```diff
@@ -191,7 +191,7 @@ class ServiceVMDetailsView(VMDetailsEntities):
     @property
     def is_displayed(self):
         return (
-            self.in_myservices and self.myservice.is_opened and
+            self.myservice.is_opened and
```

Comment:
Anything you can validate against to replace the call to `self.in_myservices`?

Reply:
ServiceVMDetailsView(VMDetailsEntities) 
ServiceVMDetailsView is inherting VMDetailsEntities which does not have in_myservice.

So can't implement self.in_myservices